Appraisals

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+appraise 'rails-4.2' do
+  gem 'sqlite3', '~> 1.3.6'
+  gem 'activerecord', '4.2.11.1'
+  gem 'activesupport', '4.2.11.1'
+end
+
+appraise 'rails-5.0' do
+  gem 'sqlite3', '~> 1.3.6'
+  gem 'activerecord', '5.0.7.2'
+  gem 'activesupport', '5.0.7.2'
+end
+
+appraise 'rails-5.1' do
+  gem 'activerecord', '5.1.7'
+  gem 'activesupport', '5.1.7'
+end
+
+appraise 'rails-5.2' do
+  gem 'activerecord', '5.2.3'
+  gem 'activesupport', '5.2.3'
+end
+
+appraise 'rails-6.0' do
+  gem 'activerecord', '6.0.0.rc2'
+  gem 'activesupport', '6.0.0.rc2'
```

Comment:
Are you going to bump this when 6.0 final is released? Any sense in holding out until then?

Reply:
Yup. I'll bump it when Rails 6.0.0 is released. Rails has been in RC since April 24, 2019. I'm not sure what the hold up is but I'm doing some more proactive testing of Rails 6.0 in activerecord-extensions and this unblocks that effort.